data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -386,11 +386,13 @@
     "id": "guns_rifle_rare_display",
     "//": "Less common rifles found exclusively in gun stores.",
     "items": [
+      { "item": "m16_auto_rifle", "variant": "acr", "prob": 25, "charges-min": 0, "charges-max": 0 },
       { "item": "colt_lightning", "prob": 15, "charges-min": 0, "charges-max": 0 },
       { "item": "m249_semi", "prob": 5, "charges-min": 0, "charges-max": 0 },
       { "item": "henry_big_boy", "prob": 10, "charges-min": 0, "charges-max": 0 },
       { "item": "m14ebr", "prob": 15, "charges-min": 0, "charges-max": 0 },
       { "item": "M24", "prob": 15, "charges-min": 0, "charges-max": 0 },
+      { "item": "m4_carbine", "variant": "m4a1", "prob": 45, "charges-min": 0, "charges-max": 0 },
```

Comment:
```suggestion
```

Reply:
took a minute for me to realize what was wrong with those guns being on the list

Reply:
Sorry, I should have left a comment.

Reply:
No problem, I should've read what they were under more carefully